e2e_test/ch-benchmark/q17.slt.part

```diff
@@ -0,0 +1,11 @@
+query R rowsort
+select	sum(ol_amount) / 2.0 as avg_yearly
+from    orderline, (select   i_id, avg(ol_quantity) as a
+		            from     item, orderline
+		            where    i_data like '%b'
+		            	     and ol_i_id = i_id
+	        	    group by i_id) t
+where   ol_i_id = t.i_id
+	    and ol_quantity < t.a;
+----
+NULL
```

Comment:
Will we change the data a bit to let q17 produce a number?

Reply:
I've updated the input data so that now q17 does produce some output. During testing with the new data I encountered a problem with q19. It produces a slightly different outcome due to loss of precision on each execution, so I've deleted q19 for now.

Reply:
I see, is it possible to use `round` to bypass the` loss of prevision` problem? Or the difference can actually be quite large.

Reply:
The issue with q19 is that the output is inconsistent (which I assume is caused by the loss of precision).

The query:

```
select  sum(ol_amount) as revenue
    from  orderline, item
    where       (
              ol_i_id = i_id
              and i_data like '%a'
              and ol_quantity >= 1
              and ol_quantity <= 10
              and i_price between 1 and 400000
              and ol_w_id in (1,2,3)
            ) or (
              ol_i_id = i_id
              and i_data like '%b'
              and ol_quantity >= 1
              and ol_quantity <= 10
              and i_price between 1 and 400000
              and ol_w_id in (1,2,4)
            ) or (
              ol_i_id = i_id
              and i_data like '%c'
              and ol_quantity >= 1
              and ol_quantity <= 10
              and i_price between 1 and 400000
              and ol_w_id in (1,5,3)
            );
```

Executing the query results in one of the following outcomes (pg always results in `15975.904`):

```
15975.901
15975.902
15975.903
15975.904
```

So yeah rounding the value to 2 decimals would at least result in a consistent outcome.